jest.config.js

```diff
@@ -1,24 +1,18 @@
 module.exports = {
```

Comment:
The PolkadotJS jest config:
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/jest.js

Reply:
Well, the jest config I have is inherited from my vuejs template; added a note to #34 to revisit this config (and probably borrow the one from polkadot-js) when removing the VueJS